core-services/prow/02_config/openshift/cloud-provider-kubevirt/_prowconfig.yaml

```diff
@@ -0,0 +1,114 @@
+tide:
+  queries:
+  - includedBranches:
```

Comment:
Do all of these branches actually exist? If they don't exist, do we need them in this list?

Reply:
Thank you @dhellmann 
This was actually auto generated and I'm not sure if I need or can change it
@droslean can you please advise on this?